openstack/compute/v2/extensions/quotasets/requests.go

```diff
@@ -11,6 +13,13 @@ func Get(client *gophercloud.ServiceClient, tenantID string) GetResult {
 	return res
 }
 
+// GetDetail returns detailed public data about a previously created QuotaSet.
+func GetDetail(client *gophercloud.ServiceClient, tenantID string) GetResultDetail {
+	var res GetResultDetail
+	_, res.Err = client.Get(getURL(client, fmt.Sprintf("%s/detail", tenantID)), &res.Body, nil)
```

Comment:
I think this would look cleaner if there was a url (`getDetailURL`) added to `urls.go`